specification/personal-demographics.yaml

```diff
@@ -1057,26 +1057,63 @@ paths:
         * `/address/0` if the address to be replaced or removed is the first item in the list. If it is the second item in the list the path will be `/address/1`, and so on.
         * `/address/-` when adding a new address.
 
-        An Address consists of 5 lines of unstructured text, postcode and PAF (Postcode Address File) key.
+        An address consists of 5 lines of unstructured text, postcode and PAF (Postcode Address File) key.
         Local systems must apply the following formatting convention when adding or replacing addresses lines:
-        *	Line 1 - premises ID and/or house name, e.g. `Flat 1` or `The Old Schoolhouse`
-        *	Line 2 - house number, dependent thoroughfare name and descriptor (if present), thoroughfare name and descriptor, e.g. `23 Mill Lane`
-        *	Line 3 - dependent locality/village, locality (if present), e.g. `Boxgrove`
-        *	Line 4 - post town, e.g. `Leeds`
-        *	Line 5 - county (if present), e.g. `West Yorkshire`.
+        *	line 1 - premises ID and/or house name, e.g. `Flat 1` or `The Old Schoolhouse`
+        *	line 2 - house number, dependent thoroughfare name and descriptor (if present), thoroughfare name and descriptor, e.g. `23 Mill Lane`
+        *	line 3 - dependent locality/village, locality (if present), e.g. `Boxgrove`
+        *	line 4 - post town, e.g. `Leeds`
+        *	line 5 - county (if present), e.g. `West Yorkshire`
 
-        When updating an address, lines 1 or 2, and line 4 must be populated; line 5 should not be included in vernacular addresses but may be included in PAF-derived addresses.
+        When updating an address, lines 1 or 2, and line 4 should be populated; line 5 should not be included in manually created addresses but may be included in PAF derived addresses.
+
+        Address lines should be populated otherwise the message may be rejected. There are exceptions:
+        * if a postcode and PAF key are submitted, in which case the lines will be automatically populated, however if there are no matches or too many matches the message will be rejected due to missing address lines
+        * a pseudo postcode is used. For example `ZZ99 3VZ` meaning `no fixed abode`
```

Comment:
Is there a link we can add in here to the list of peusdo postcodes

Reply:
Links in same document don't work in APIGEE